File: Benchmarks/FizzBuzz.cs
namespace LeetCode;
using BenchmarkDotNet.Attributes;

public partial class Benchmarks
{
    [Benchmark]
    public void FizzBuzz()
    {
        Submission.FizzBuzz(3);
        Submission.FizzBuzz(5);
        Submission.FizzBuzz(15);
    }
}

File: LeetCode.FSharp/Benchmarks/FizzBuzz.fs
namespace LeetCode.FSharp.Benchmarks

open BenchmarkDotNet.Attributes
open LeetCode.FSharp.Problems

[<Config(typeof<LeetCode.CSharp.RuntimeConfig>)>]
type FizzBuzz() =

    [<Benchmark>]
    member _.FizzBuzz() = [
        FizzBuzz.FizzBuzz 1_000 ];
